core/modules/views_ui/views_ui.admin.inc

```diff
@@ -2829,7 +2846,7 @@ function views_ui_ajax_form($js, $key, $view, $display_id = '') {
   array_splice($args, 0, 4);
 
   $form_state = views_ui_build_form_state($js, $key, $view, $display_id, $args);
-  // check to see if this is the top form of the stack. If it is, pop
+  // Check to see if this is the top form of the stack. If it is, pop
```

Comment:
You seem to have been fixing some of the comments in this file, but this one seems to have some weird wrapping. Looks like it could fit in two lines if properly wrapped.

Having said that, I would leave the comment fixing out of this PR. We will handle them in bulk as part of https://github.com/backdrop/backdrop-issues/issues/3213

Reply:
done.